distribution/docs/src/main/resources/content/_tables/ContentDirectoryMonitor.adoc

```diff
@@ -5,7 +5,7 @@
 :application: ${ddf-catalog}
 :summary: Content Directory Monitor configurations.
 
-.[[org.codice.ddf.catalog.content.monitor.ContentDirectoryMonitor]]Catalog Content Directory Monitor
+.[[_org.codice.ddf.catalog.content.monitor.ContentDirectoryMonitor]]Catalog Content Directory Monitor
```

Comment:
❓ We refer to this by `{application-prefix}org.codice.ddf.catalog.content.monitor.ContentDirectoryMonitor` in this PR. Should this have the `{application-prefix}`?

Reply:
only the `<<>>` anchors should need the prefix